### DISCUSSION TITLE
What is the support plan for 1.4 now 2.0 is out?

### DISCUSSION THREAD

**Discussion by mattseymour** [sqlalchemy/sqlalchemy#10023] [Usage Questions]
I could not see an answer on the webstie or in the issuse list, but whats the plan for supporting v1.4 of SQL Alchemy going forwards now that sql alchemy 2 is out.

Is there a support window which will be applied?

**Comment by CaselIT** [comment]
Hi,

The site talks a bit about this https://www.sqlalchemy.org/download.html#relstatus
1.4 is in maintenance mode, it should be so until 2.1 is out.

Work for 2.1 has not yet started, usually there are a couple of years between major releases.

Was your question more specific on something? Note that it should not be too hard migrating from 1.4 to 2.0, since most of the v2 api is present also on 1.4
